crates/nu-cli/src/completions/operator_completions.rs

```diff
@@ -40,57 +40,57 @@ impl Completer for OperatorCompletion {
         };
         let possible_operations = match op {
             Expr::Int(_) => vec![
-                ("+", "Plus / Addition"),
-                ("-", "Minus / Subtraction"),
+                ("+", "Add (Plus)"),
+                ("-", "Subtract (Minus)"),
                 ("*", "Multiply"),
                 ("/", "Divide"),
-                ("==", "Equal"),
-                ("!=", "Not Equal"),
-                ("//", "Floor Division"),
-                ("<", "Less Than"),
-                (">", "Greater Than"),
-                ("<=", "Less Than or Equal to"),
-                (">=", "Greater Than or Equal to"),
-                ("mod", "Modulo"),
-                ("**", "Pow"),
-                ("bit-or", "bitwise or"),
-                ("bit-xor", "bitwise exclusive or"),
-                ("bit-and", "bitwise and"),
-                ("bit-shl", "bitwise shift left"),
-                ("bit-shr", "bitwise shift right"),
+                ("==", "Equal to"),
+                ("!=", "Not equal to"),
+                ("//", "Floor division"),
+                ("<", "Less than"),
+                (">", "Greater than"),
+                ("<=", "Less than or equal to"),
+                (">=", "Greater than or equal to"),
+                ("mod", "Floor division remainder (Modulo)"),
+                ("**", "Power of"),
+                ("bit-or", "Bitwise OR"),
+                ("bit-xor", "Bitwise exclusive OR"),
+                ("bit-and", "Bitwise AND"),
+                ("bit-shl", "Bitwise shift left"),
+                ("bit-shr", "Bitwise shift right"),
             ],
             Expr::String(_) => vec![
-                ("=~", "Regex Match / Contains"),
-                ("!~", "Not Regex Match / Not Contains"),
-                ("in", "In / Contains (doesn't use regex)"),
+                ("=~", "Contains regex match"),
+                ("!~", "Does not contain regex match"),
+                ("in", "In / Contained by (no regex)"),
```

Comment:
I'd rather have "Contains" here because "Contained by" makes me think too much about what it means. Also, I like the "doesn't use regex" better than "no regex" because it's more self-explanatory.

Reply:
I understand the concern with *"no regex"* vs. *"doesn't use regex"* and will change it back, but are you sure about *"Contains"* - That's actually the opposite of what it does.

```nu
"a" in [ a b c ]
```

That doesn't mean, *"a contains a, b, or c"*.

What about, *"Is a member of (doesn't use regex)"*?  

*"a is a member of [ a b c ]"*

Reply:
Or just *"In / Member of (doesn't use regex)"*?

Reply:
ya, "is a member" of sounds better. thanks!